lizzy-eclipse/lizzy-eclipse-updatesite/deployUpdatesite.sh

```diff
@@ -7,11 +7,19 @@ rm latest/artifacts.jar
 rm latest/content.jar
 # login to intension server host:
 ncftp -u $1 -p $2 wp12720414.server-he.de <<END_SCRIPT
+# get previous version:
+prev=$(cat latest/release-number.txt)
+# copy old jars:
+mkdir release-$prev
+cp -r latest/ release-$prev/
 # delete old jars:
 delete latest/features/*
 delete latest/plugins/*
```

Comment:
TODO : don't copy dir and delete old jars - instead: `mv` dir `latest` to new dir `release-$prev`